packages/core/components/FileDetails/FileDetails.module.css

```diff
@@ -158,8 +159,9 @@
     border: none;
     color: var(--primary-text-color);
     height: 30px;
-    max-width: 50%;
+    max-width: var(--max-detail-button-width);
```

Comment:
Could we also keep these aligned center? E.g., in `.file-actions {`, line 104 `justify-content: center;`

Reply:
resolved in [43cb537](https://github.com/AllenInstitute/aics-fms-file-explorer-app/pull/103/commits/43cb537ff521b7f5d90dbb268ae3525878022f9e)